### PR TITLE
tests: fix print_status check, in some case log may not flushed to disk

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 
fix occasional CI failed, such as https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/dm_ghpr_test/detail/dm_ghpr_test/726/pipeline

after dig into log, I find syncer log is not flushed to disk during test running in time.

### What is changed and how it works?
1. disable heartbeat in `print_status` test, so we will have determined binlog event counts
2. add retry mechanism in syncer print status check
3. **update DM-worker, DM-master startup and check rpc available sequence** to following
```
start DM-worker1
check DM-worker1 rpc service is available
start DM-worker2
check DM-worker2 rpc service is available
start DM-server
check DM-server rpc service is available
```

btw, why there are 92 binlog events in test case but there exists 94 SQL statements, this is because the following two statements change zero rows.
```
UPDATE `t_1` SET `c12` = 'a' WHERE `id` = 50;
UPDATE `t_1` SET `c12` = 'c' WHERE `id` = 87;
```

### Check List 

Tests 

 - Integration test

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@amyangfei is this PR ready for review now.

**Comment by amyangfei** [comment]
> @amyangfei is this PR ready for review now.

it is ready for review now, but I will make several times of `run-all-test` to see if result is stable

**Comment by amyangfei** [comment]
PTAL @GregoryIan @csuzhangxc

File: tests/print_status/run.sh [hunk 1]
```diff
@@ -50,6 +52,23 @@ function run() {
     check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
 
     # check sync unit print status
+    i=0
+    while [ $i -lt 3 ]
+    do
+        event_count=$(grep "receive binlog event with header" $WORK_DIR/worker1/log/dm-worker.log|grep -v relay|grep "XIDEvent"|wc -l)
```
Comment (csuzhangxc) [inline tests/print_status/run.sh]: grep from _debug_ level log?

Comment (amyangfei) [reply tests/print_status/run.sh]: yep, the log level in integration test is DEBUG

Comment (csuzhangxc) [reply tests/print_status/run.sh]: OK. but notice the change of the log.

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**